source/_posts/2017/2017-06-17-php-object-calisthenics-rules-made-simple-version-3-0-is-out-now.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "PHP Object Calisthenics Made Simple - Version 3.0 is Out Now"
+perex: '''
```

Comment:
Not yet, this is the first post here.
There is a link in first paragraph to external source.